.github/workflows/continuous-integration.yaml

```diff
@@ -74,11 +74,4 @@ jobs:
           go-version: 1.17
       - name: Run api-docs
         run: |
-          make ensure-generate-is-noo api-docs
-      - name: Check if working tree is dirty
-        run: |
-          if [[ $(git status --porcelain) ]]; then
-            git diff
-            echo 'run make api-docs and commit changes'
-            exit 1
-          fi
+          make api-docs
```

Comment:
Almost! If you add `api-docs` as dependency for the `ci` job, you don't need to change this workflow at all :-)

Reply:
That's true. Good catch! Thanks